changelog.txt

```diff
@@ -1,6 +1,7 @@
 0.84.1
  * Fixed a bug where simultaneous opening and closing a Realm on different threads might result in a NullPointerException (#1646).
  * Suppressed warnings reported by `lint -Xlint:all` (#1644).
+ * Updated ProGuard configuration. See official document https://realm.io/docs/java/latest/#proguard for details.
```

Comment:
_See documentation https://realm.io/docs/java/latest/#proguard for more details_